qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, controller, configuration, provider=None):
         not available.
 
         Args:
-            controller (function): Aer cython controller to be executed
+            controller (function): Aer controller to be executed
```

Comment:
This doesn't seem related to the asv changes

Reply:
Yep, that was an improvement I just did for free xD